python_modules/dagster/dagster_tests/definitions_tests/asset_reconciliation_tests/auto_observe_scenarios.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+
+import pendulum
+from dagster import DataVersion, observable_source_asset
+
+from .asset_reconciliation_scenario import (
+    AssetReconciliationScenario,
+    run_request,
+)
+
+
+@observable_source_asset(auto_observe_interval_minutes=30)
```

Comment:
should we have a test with a partitioned observable source asset?

Reply:
good idea - adding. shouldn't actually behave any differently than the single asset scenario, but good to encode that.